.github/workflows/maven.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Deploy P2 Updatesite
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.11
+    - name: Build with Maven
+      run: mvn -B package --file pom.xml
+    - name: Replace Updatesite
+      run: |
+        rm -r updatesite
+        git config --local user.email "tellastory73@gmail.com"
+        git config --local user.name "Ingo Mohr"
+        git checkout -B updatesite
+        git rm -r updatesite/
+        mkdir updatesite
+        cd releng-updatesite/target
+        mv  -v repository/* ../../updatesite/
+        cd ..
+        cd ..
+        git add updatesite/
+    - name: Commit new Updatesite
+      run: |
+        git commit -m "deploy new updatesite"
```

Comment:
A cool (but totally unnecessary) improvement for a future PR would be to read the pom.xml version and reference it in the commit message, like "deploy new updatesite for 1.0.8". Just thinking outloud :) It might be too annoying/hard for the value it gives.

Reply:
Yess! 🔥 Very good idea. I‘ll check that. 😊

Reply:
I would like to discuss this further in its own issue. What do you think, @AObuchow?
This could make sense once we have a build script that adds new versions instead of simply replacing the latest version.

Reply:
For sure :) Feel free to make an issue for it, if not I'll remember to make one